marklogic-data-hub-central/ui/src/pages/Browse.tsx

```diff
@@ -132,13 +136,15 @@ const Browse: React.FC<Props> = ({ location }) => {
     return () => {
       componentIsMounted.current = false
     }
-  }, [])  
+  }, [])
 
   useEffect(() => {
     if (entities.length && (!searchOptions.nextEntityType || searchOptions.nextEntityType === 'All Entities' || (searchOptions.entityTypeIds[0] == searchOptions.nextEntityType))) {
-    getSearchResults(entities);
-    }
-  }, [searchOptions, searchOptions.zeroState === false && entities, user.error.type, database]);
+      // if ((searchOptions.entityTypeIds[0] === searchOptions.nextEntityType) || (!searchOptions.entityTypeIds[0] && (searchOptions.nextEntityType === 'All Entities' || searchOptions.nextEntityType === 'All Data'))) {
```

Comment:
Let's avoid keeping the previous code as a comment, if we have replaced it with new one, unless the comment is relevant to maintain the code piece and provide more information to other developers.

Reply:
Removed.